### ISSUE TITLE
Search box in extensions panel was zoomed with mouse wheel zooming in editor. 

### ISSUE THREAD

**Issue by chen86860** [microsoft/vscode#58784]
- OS Version: macOS High Sierra 10.13.6

Steps to Reproduce:

1. Search box in extensions panel was zoomed with mouse wheel zooming in editor. 

Does this issue occur when all extensions are disabled?: Yes

---

When I try searching  extensions(eg: Docker), the search box was zoomed with zooming in editor. it shouldn't be. 

![search box was zoomed with editor](https://i.loli.net/2018/09/17/5b9f57b7ba058.png)

**Comment by ramya-rao-a** [comment]
@chen86860 Can you provide exact steps to reproduce this issue? I've tried zooming using Cmd+, changing font size, but I still dont see it.

Are you able to consistently see this? i.e window reload doesnt fix it?

Can you share your settings?

**Comment by roblourens** [comment]
It breaks the breakpoint editor too

![image](https://user-images.githubusercontent.com/323878/45654821-39a66d80-ba92-11e8-8902-1fa62811a75b.png)

**Comment by roblourens** [comment]
@rebornix do you know how to disable this for the simple editors?

**Comment by ramya-rao-a** [comment]
@rebornix Setting `mouseWheelZoom` to `false` in the editor options doesnt fix it.

**Comment by shroudedcode** [comment]
I just opened an issue about this and found this duplicate. Since I closed that one, here's the GIF I created for it. It shows show how the font updates while zooming and that the placeholder text is not affected by that.

![](https://user-images.githubusercontent.com/29176678/48505015-608ce000-e846-11e8-9855-4d678c43d845.gif)

I've been using the *Mouse Wheel Zoom* feature for a long time now and this bug is just so annoying. For a while I thought it was caused by some extension or weird settings but now I'm experiencing it again on a fresh installation with only these settings:

```json
{
    "editor.mouseWheelZoom": true,
    "diffEditor.renderSideBySide": false,
    "workbench.statusBar.feedback.visible": false,
    "window.titleBarStyle": "custom",
    "window.zoomLevel": 0
}
```

**Comment by roblourens** [comment]
Investigated a little more and one thing I found is that setting `mouseWheelZoom` on the simple editors does prevent it from working on those editors, but the editor zoom level is shared between all instances of editors, and it doesn't prevent that. 

The editor zoom is used for mousewheel zoom and font zoom. We would want to disable both of these for the simple editors. There is no way to disable font zoom currently. How about we add another option to `IEditorOptions` which disables EditorZoom for an editor instance entirely?

**Comment by chen86860** [comment]
I found that the bug was fixed after version `1.35.1` (I have no clear which version it started) on macOS, so I close this issue. Thanks you guys~ 😃
